packages/loader/container-loader/src/container.ts

```diff
@@ -2461,11 +2465,19 @@ export class Container
 			snapshot,
 		);
 
-		this._runtime = await PerformanceEvent.timedExecAsync(
+		const runtime = await PerformanceEvent.timedExecAsync(
 			this.subLogger,
 			{ eventName: "InstantiateRuntime" },
 			async () => runtimeFactory.instantiateRuntime(context, existing),
 		);
+
+		const maybeRuntimeCompatDetails = runtime as unknown as FluidObject<ILayerCompatDetails>;
```

Comment:
I think this works?

```suggestion
		const maybeRuntimeCompatDetails = runtime as FluidObject<ILayerCompatDetails>;
```

Reply:
It wasn't working that's why I had to add unknown. But it's working now! Don't know what changed.